test/gtest/possible-contents.cpp

```diff
@@ -74,223 +74,215 @@ static std::unique_ptr<Module> parse(std::string module) {
 
 // We want to declare a bunch of globals that are used in various tests. Doing
 // so in the global scope is risky, as their constructors use types, and the
-// type system itself uses global constructors. Instead, put all the tests
-// inside a "testMain()" function whose scope will contain those reused
-// "globals."
-TEST(PossibleContentsTests, PossibleContentsTest) {
-  // Use nominal typing to test struct types.
-  wasm::setTypeSystem(TypeSystem::Nominal);
-
-  auto none_ = PossibleContents::none();
-
-  auto i32Zero = PossibleContents::literal(Literal(int32_t(0)));
-  auto i32One = PossibleContents::literal(Literal(int32_t(1)));
-  auto f64One = PossibleContents::literal(Literal(double(1)));
-  auto anyNull = PossibleContents::literal(Literal::makeNull(HeapType::any));
-  auto funcNull = PossibleContents::literal(Literal::makeNull(HeapType::func));
-
-  auto i32Global1 = PossibleContents::global("i32Global1", Type::i32);
-  auto i32Global2 = PossibleContents::global("i32Global2", Type::i32);
-  auto f64Global = PossibleContents::global("f64Global", Type::f64);
-  auto anyGlobal = PossibleContents::global("anyGlobal", Type::anyref);
-
-  auto func = PossibleContents::literal(
+// type system itself uses global constructors. Instead, we use a fixture for
+// that.
+class PossibleContentsTest : public testing::Test {
+protected:
+  void SetUp() override {
+    // Use nominal typing to test struct types.
+    wasm::setTypeSystem(TypeSystem::Nominal);
+  }
+
+  PossibleContents none_ = PossibleContents::none();
```

Comment:
Do we need the underscore here?
```suggestion
  PossibleContents none = PossibleContents::none();
```

Reply:
Ah, good point, this is not needed since the `none` type is namespaced in `wasm::Type`, so no conflict... fixing.